framework/src/vectorpostprocessors/SpatialUserObjectVectorPostprocessor.C

```diff
@@ -28,6 +28,8 @@ SpatialUserObjectVectorPostprocessor::validParams()
                             "A filename that should be looked in for points. Each "
                             "set of 3 values in that file will represent a Point.  "
                             "This and 'points' cannot be both supplied.");
+  params.addParam<bool>(
+      "use_points_from_uo", false, "Whether to obtain the points directly from the user object");
```

Comment:
Can you make it such that if neither `points` or `points_file` are specified, the code will go use points from the UO?

Reply:
Yeah that's a good idea. I also just realized I should probably do the `NearestPointLayeredSide...` objects, too.